lib/telephony/pinpoint/voice_sender.rb

```diff
@@ -4,18 +4,18 @@
 module Telephony
   module Pinpoint
     class VoiceSender
-      ClientConfig = Struct.new(:client, :config)
-
       # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/BlockLength
       def send(message:, to:)
-        return handle_config_failure if client_configs.empty?
+        return handle_config_failure if Telephony.config.pinpoint.voice_configs.empty?
 
         language_code, voice_id = language_code_and_voice_id
 
         last_error = nil
```

Comment:
I think this would be similar to the default `response` in the SMS Sender, and above comment would apply to this as well

Reply:
81f3063